docs/_documentations/troubleshooting.md

```diff
@@ -363,6 +364,49 @@ These steps reproduce the issue:
 
 **Workaround** Run the `Attach Debugger` action manually.
 
+***
+# ODO with Codewind
+
+<!--
+Codewind version: 0.5.0
+Che version: 7.2.0
+IDE extension version: 0.5.0
+IDE version: **7.1.0
+Action/Topic: ODO with Codewind
+Issue type: bug/info
+Issue link: https://github.com/eclipse/codewind/issues/692
+-->
+## ODO projects are not deleted after the workspace is deleted
+
+If you create ODO projects and then delete the workspace without deleting the projects, all your ODO deployments still exist. 
+
+Follow these steps to reproduce the issue: 
```

Comment:
Thinking we don't need the `steps to reproduce the issue` section?

Reply:
Have reduced to `These steps to reproduce the issue:` as a lead in sentence, which is more common amongst all the issues documented in this file.